devimint/src/federation.rs

```diff
@@ -203,6 +218,13 @@ impl Federation {
         Ok(invite_code)
     }
 
+    pub fn invite_code_for(&self, peer_id: u16) -> Result<String> {
```

Comment:
`self` not used?

Reply:
Should `Federation::invite_code(&self) -> Result<String>` be changed to `Federation::invite_code() -> Result<String>` as well? This method is already being used and all of the places it is used would need to be changed since it would become an associated function instead of a method without `self`